src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -31,9 +31,13 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
   };
 
   const uploadFile = async (e: any) => {
+      const token = localStorage.getItem('authorization_token')
+      const headers = token ? {'Authorization':token} : {}
```

Comment:
Hi, it is contraversial topic. But I expected to have here 'Balic ${token}'. So it took me time to understand how to test your app.
It could be better to write down the way your app should be tested - in this case, if you include 'Basic' in local storage.
Just a note for future)